core/CronArchive.php

```diff
@@ -460,6 +462,12 @@ public function run()
         );
 
         $this->logger->info($timer->__toString());
+        $this->logger->info("Mermory peak usage: " . $this->getHumanReadableBytesSize(memory_get_peak_usage()));
+    }
+
+    private function getHumanReadableBytesSize($memoryInBytes, $precision = 2) {
```

Comment:
Could you please format the code consistently with our code / PSR notation, ie add line breaks and spacing